_layouts/app.html

```diff
@@ -25,6 +25,12 @@ <h5 class="h6">Developer</h5>
         <h5 class="h6">About</h5>
         <div class="label">{{ content }}</div>
       </div>
+    </div>
+     <div class="meta-row">
+      <div class="div-block-3">
+        <h5 class="h6">Donate</h5>
+        {% if page.donate %} • <a href="{{ page.donate }}">donate</a>{% endif %}
```

Comment:
With this setup all apps will have a "Donate" block but no content when no link is set.
Can't you put the if at the block level so it's not just empty content for most apps?

Also what is " • " doesn't it break alignment with the rest of the content?